### ISSUE TITLE
bug: extension makes intellisense behaviour slow

### ISSUE THREAD

**Issue by roopkt** [buehler/typescript-hero#388]
<thank you for using and contributing to typescript hero>
<please fill the information below to ease up the process>
<you may even implement it and send a PR which would be very appreciated>
I love typescript hero However when I upgraded my VsCode to 64 bit and used 2.1.1 typescript hero then it is very very slow. It is showing me the intellisence very late and once I select option then it changes the document after 5 to 7 seconds of delay.
#### System information

- OS:  Windows 2010
- TypeScript Hero version: 2.1.1
- VSCode version: 1.19.3 ( 64 bit )

#### Description
it is very very slow. It is showing me the intellisence very late and once I select option then it changes the document after 5 to 7 seconds of delay.
<what is going wrong>
displaying intellisence very late and updating file after long delay
##### Steps to reproduce

Add any class name in your file that you want to import initially it used to import automatically once I write class name and hit enter now it is doing that but after long delay. 
Also if you hoverover on class name it will show intellisence to import bulb very late. 
<for feature requests, please use the following template>

#### Feature <name>

<description>

#### Reason

<why it would be cool>

**Comment by buehler** [comment]
Hey @roopkt 
That's a strange occurrence. Actually, there is _no_ intellisense right now. It was removed, so this must be another extension that is causing this problem. The actual intellisense you are encountering is the one of the normal typescript compiler.

**Comment by adraax** [comment]
I have the same problem on macOs 10.13.2, VS 1.19.3 and TypeScript Hero 2.1.1.

When this extension is activated, the base intellisense take at least 5 second to show a suggestion, after uninstalling it, intellisense suggestion show up with no delay.

As @roopkt said, it can be easily reproduced with an import, like ```import { BrowserModule } from '@angular/platform-browser';``` for example.

**Comment by roopkt** [comment]
Thanks @buehler  for responding may be vscode intellisense is slow. Because I do not have any other extension.  I only have 4 extensions 1) Angular language service 2) Vscode icons 3) TsLint 4) Typescript Hero
But I am surprised why we have removed the import namespace feature. I was using typescript hero only for that reason when I type class name and hit enter it used to pull the import with accurately and immediately . However now that feature is no more which is sad. I used to use ALT+CTL+SHIFT I to import. At this point I am thinking there is no more use of Typescript Hero.
